module/cli_controller.py

```diff
@@ -118,7 +89,7 @@ def check_input(self) -> None:
         if not self.input_thread.isAlive() and self.input_queue.empty():
             self.start_thread()
         elif not self.input_queue.empty():
```

Comment:
Shouldn't here be a check whether the thread is alive?

Reply:
No, we don't want the thread to ask for input when we could be printing things.